fmn/api/distgit.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+import requests
```

Comment:
This is synchronous, it would be better to use httpx's `AsyncClient`.  See `fmn/api/auth.py` for an example.